build.gradle

```diff
@@ -69,6 +69,18 @@ task installRealmJava(type:Task) {
     description = 'Install the Realm library and Gradle plugin into mavenLocal()'
 }
 
+task installRobolectric(type:GradleBuild) {
+    description = 'Install the Robolectric library into example'
```

Comment:
@dalinaum right. `0.95.0` is installed in `/usr/local/include/`.

How do I update that?

/usr/local/include/realm/version.hpp
```
#define REALM_VER_MAJOR 0
#define REALM_VER_MINOR 95
#define REALM_VER_PATCH 0
```